.github/workflows/unit-tests.yml

```diff
@@ -74,6 +74,16 @@ jobs:
       - name: Setup and run tests
         uses: ./.github/actions/unit-tests
 
+      - name: Renaming coverage data file
+        run: |
+          mv reports/.coverage reports/${{ matrix.shard_name }}.coverage
```

Comment:
The .coveragerc file has this line:
```
[run]
data_file = reports/${TEST_SUITE}.coverage
```
TEST_SUITE used to be defined in the Jenkins job.  If we define it here, we don't need to move files around.  The data file is being written to ".coverage" only because TEST_SUITE isn't defined.